### DISCUSSION TITLE
Does the module commerce has a multi vendor option?

### DISCUSSION THREAD

**Discussion by WebCrew** [friends-of-forkcms/fork-cms-module-commerce#37] [Q&A] [answered]
Hallo **Community**,

I am looking for a smaller free CMS with a _multi vendor shop_ / commerce option to sell modules, plugins and themes for a other SN-CMS.

I  gave Joomla, Magento and WordPress (and the other well known big players) a try, but hey they are all a way to bloated or you need additional premium extensions and become poor because of it.

Then I came across Fork CMS and I love it so far - it is not that big but has most options I need. It would be even more awesome if it has a multi vendor option.

I would be very happy if you could inform me about this. Or maybe you know a other smaller Open Source CMS that offer the option I'm looking for?

kind regards, Andy :)

**Comment by jessedobbelaere** [comment, marked as answer]
Hi Andy, the commerce module is still a work in progress (not used in actual projects), so a more advanced feature like multi vendor is still far away unfortunately

**Reply by WebCrew** [reply]
Thank You for the info.
